conf/distro/include/tcmode-external-sourcery.inc

```diff
@@ -15,6 +15,7 @@ EXTERNAL_CROSS_NOPSEUDO = "gcc g++ cpp"
 baselib = "${@d.getVar('BASE_LIB:tune-' + (d.getVar('DEFAULTTUNE') or 'INVALID')) or d.getVar('BASELIB')}"
 
 # Ensure that the licensing variables are available to the toolchain.
+export SALT_LICENSE_SERVER
 export MGLS_LICENSE_FILE
 export LM_LICENSE_FILE
```

Comment:
Are we expected to be backwards compatible i.e. support for MGLS/LM_LICENSE_FILE?

Reply:
@Noor-Ahsan @jeremy-robinson ?

Reply:
Understood, thanks @tmirza